### DISCUSSION TITLE
Download assets from private Github releases

### DISCUSSION THREAD

**Discussion by vcrobe** [community/community#33838] [Apps, API and Webhooks] [answered]
Is it possible to download assets from a private Github releases without having access to the source code?

I need to give access to some users in my organization to download assets from a private repo but they can't see anything else in the repo (source code, issues, etc). 

Is this possible?

**Comment by lx78WyY0J5** [comment, marked as answer]
https://github.com/marketplace/actions/repository-permission
https://github.com/marketplace/actions/manage-github-teams-permission

Check in [marketplace ](https://github.com/marketplace) for this kind of perm 

You can also use Github actions to push the build file into somewhere else 

🆙 me if helped u ❤️
